example_site/src/components/Collections.js

```diff
@@ -31,45 +32,124 @@ function Template({ template }) {
   );
 }
 
+function DurationFilter({
+  filter,
+  templateFilter,
+  setTemplateFilter,
+}) {
+  const [buttonStatus, setButtonStatus] = useState(true);
+  const buttonColor = templateFilter.duration === filter.value ? theBlue : 'black';
+
+  const onSelectFilter = (newDuration) => {
+    if (buttonStatus) {
+      setTemplateFilter((currentFilter) => (
+        {...currentFilter, duration: newDuration.value}
+      ))  
+    } else {
+      setTemplateFilter((currentFilter) => {
+        const newFilters = {...currentFilter};
+        delete newFilters.duration;
+        return newFilters;
+      })
+    }
+  }
+
+  return (
+    <div className="category-name">
+      <button 
+        className="filter-name"
+        onClick={() => {
+          onSelectFilter(filter)
+          setButtonStatus(!buttonStatus)
+      }}>
+        <div style={{'color': buttonColor}}>
+          {filter.displayName} 
+        </div>
+      </button>
+    </div>
+  );
+}
+
+function AspectRatioFilter({
```

Comment:
This and the `DurationFilter` component are identical except for the fact that they're dealing w/ aspect ratio vs. duration. Let's make one reusable component!